src/main.c

```diff
@@ -41,6 +41,7 @@
 #include <shmem.h>
 #include <utils.h>
 #include <worker.h>
+#include <memory.h>
```

Comment:
Sorting.

`pgagroal.h` first, then the rest sorted, and then the system includes sorted